environment-ci.yml

```diff
@@ -0,0 +1,11 @@
+name: rte_rrtmgp_ci
+channels:
+  - conda-forge
+  - nodefaults
+dependencies:
+  - python=3.11
+  - numpy
```

Comment:
> Why is the name `environment-ci.yml` better than `environment-noplots.yml`

Either names are fine. I didn't want to delete your conda config files, which is why I made another environment.

> What is the benefit of the `channels` entry?

It is a good idea to instruct conda from which channels to fetch your packages, e.g. `netcdf` is not available in `default` it's only available in `conda-forge`. This is true for pretty much all your packages. Channels help make your env reproducible, so IMO you should be specifying it everywhere. Docs [link](https://docs.conda.io/projects/conda/en/latest/user-guide/concepts/channels.html) to `channels`. 

---

Unrelated to the previous bit, `numpy` has been added because it is needed in `compare-to-reference.py`.